test/check-networking

```diff
@@ -133,17 +152,11 @@ class TestNetworking(MachineCase):
         # Switching off rp_filter doesn't seem to be enough in the
         # case of bonds.
 
-        iface1 = self.get_iface(m, m.add_netiface('f1'));
-        iface2 = self.get_iface(m, m.add_netiface('f2', vlan=1));
-        b.wait_in_text("#networking-interfaces", iface1)
-        b.wait_in_text("#networking-interfaces", iface2)
-        if m.os == "rhel-7":
-            # Interfaces are not auto-activated on RHEL 7.
-            b.wait_in_text("#networking-interfaces tr:contains('%s')" % iface1, "Inactive")
-            b.wait_in_text("#networking-interfaces tr:contains('%s')" % iface2, "Inactive")
-        else:
-            b.wait_in_text("#networking-interfaces tr:contains('%s')" % iface1, "bps")
-            b.wait_in_text("#networking-interfaces tr:contains('%s')" % iface2, "Configuring IP")
+        mac = "52:54:00:9e:00:F5"
+        iface1 = self.get_iface(m, m.add_netiface(mac=mac));
+        iface2 = self.get_iface(m, m.add_netiface(secondary_network=True));
```

Comment:
I've pushed a commit that removes the secondary_network stuff and just uses QEMU network monitor defined networks. It has less litter, doesn't need root permissions etc.